CMake/FindITKUtil.cmake

```diff
@@ -55,7 +55,7 @@ macro(FindITKUtil ITK_VAR_PREFIX )
     ${ARGN}
     )
   list(REMOVE_DUPLICATES ITK_INCLUSION_MODULES)
-  find_package(ITK 4.5 COMPONENTS ${ITK_INCLUSION_MODULES} REQUIRED)
+  find_package(ITK COMPONENTS ${ITK_INCLUSION_MODULES} REQUIRED)
```

Comment:
BRAINSTools is built in many different ways (i.e. as part of Slicer and other packages).  This is only one way, and we hope to keep the version minimum so that it is easier to debug elsewhere.